apps/www/registry/new-york/ui/slider.tsx

```diff
@@ -8,17 +8,29 @@ import { cn } from "@/lib/utils"
 const Slider = React.forwardRef<
   React.ElementRef<typeof SliderPrimitive.Root>,
   React.ComponentPropsWithoutRef<typeof SliderPrimitive.Root>
->(({ className, ...props }, ref) => (
+>(({ className, orientation = "horizontal", ...props }, ref) => (
   <SliderPrimitive.Root
     ref={ref}
+    orientation={orientation}
     className={cn(
-      "relative flex w-full touch-none select-none items-center",
+      "relative flex touch-none select-none items-center",
+      orientation === "vertical" ? "flex-col h-full w-[20px]" : "w-full",
```

Comment:
again, we can just use `data-*` to check orientation. And we don't really need to set default to "horizontal" here because Radix primitive slider component default is "horizontal"